### PR TITLE
7 types added

### PR DESCRIPTION
**Pull request by nileshlad09**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,18 @@ Here are some resources to technical writing interview questions to help you pre
 - [Mirror.xyz](https://mirror.xyz/) ~ Mirror.xyz is a publishing platform for writers that leverages cryptocurrency and blockchain technology.
 - [Freecodecamp](https://www.freecodecamp.org/news/how-to-write-for-freecodecamp/) ~ FreeCodeCamp is a non-profit organization that consists of an interactive learning web platform, an online community forum, chat rooms, online publications and local organizations that intend to make learning web development accessible to anyone.
 
+## 7 Types of Technical-writing
```
Comment (dharmelolar) [inline README.md:185]: @nileshlad09 can you just make it " Types of Technical Writing" and omit the 7

Also, you should capitalize the first letter of each word you listed.

Comment (nileshlad09) [reply README.md:185]: Ok

Comment (nileshlad09) [reply README.md]: @dharmelolar I made changes as you said you can check now.

**Comment by nileshlad09** [comment]
Hey can you please add hacktober-accept label

**Comment by nileshlad09** [comment]
Yep

**Pull request merged**